policy/LAM/integration_test.yaml

```diff
@@ -0,0 +1,57 @@
+#
```

Comment:
Why is this in the `policy` directory?
What does this have to do with `LAM`?

Reply:
There is no reason for this choice. Where should it be placed?

Reply:
I'm not sure. Maybe under `examples` or `etc`?

Reply:
This file has been moved to `examples/`.